README.md

```diff
@@ -30,6 +32,15 @@ Your Answer:
 
 ```js
 // your code here
+var givesHomework = function(name, assignment){
```

Comment:
We were expecting a constructor function named Instructor, with this method.  Please review our recommendation in the solution branch.